src/components/TextArea/TextArea.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import { render, fireEvent, screen } from '@testing-library/react';
+
+import CustomTextArea from './TextArea';
+
+describe('CustomTextArea', () => {
+  it('should render the component textarea', () => {
```

Comment:
evitar "should", o `it` é pra manter a imperatividade da sentença. Use "renders" e remove o "should"

Reply:
@flaviafreirec 

https://github.com/Alecell/octopost/blob/master/STYLEGUIDE.md#testes-unit%C3%A1rios